packages/block-library/src/gallery/deprecated.js

```diff
@@ -15,515 +15,502 @@ import { RichText } from '@wordpress/block-editor';
 import { defaultColumnsNumber } from './shared';
 
 const deprecated = [
-	{
```

Comment:
I was having trouble with the block invalidating when reloading editor, so just commented this out to try and narrow down cause of that, will add this back in soon and try and see how feasible a migration from existing gallery to this approach is going to be

Reply:
fixed  - column size was not being calculated correctly which was causing the invalidation